config/config_test.go

```diff
@@ -388,6 +388,35 @@ var _ = Describe("BlobstoreClient configuration", func() {
 			})
 		})
 	})
+
+	Describe("returning the OSS region", func() {
```

Comment:
I'd mention something about Alicoud here. `OSS` can be an ambiguous acronym.

Reply:
I will improve it.